src/lib.rs

```diff
@@ -405,6 +405,9 @@ mod tests {
         assert!(count > 9);
     }
 
+    // assert! has a format! macro in it, clippy gets confused about this.
```

Comment:
If assert!() doesn't follow uninlined_format_args, we should probably just add a global exception in .github/workflows/clippy.sh instead of these specific exceptions.